LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) Tom Preston-Werner
```

Comment:
Way to anxious to close my first issue. I made the fix in def974d28923660245a6ddd205a9e172aa856e4a